Polygon.Net/API/NewsApi.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Polygon.Net.Models;
+using static Pineapple.Common.Preconditions;
+
+namespace Polygon.Net;
+
+public partial class PolygonClient
+{
+    private const string NEWS_ENDPOINT = "/v2/reference/news";
+
+    /// <summary>
+    /// Get the news from Polygon.
+    /// </summary>
+    /// <param name="startTime">Return results published after this date</param>
+    /// <param name="endTime">Return results published before this date</param>
+    /// <param name="ticker">The polygon API ticker by default is desc</param>
+    /// <param name="order">Order results based on the sort field</param>
+    /// <param name="limit">Limit the number of results returned, default is 10 and max is 1000</param>
+    /// <param name="sort">Sort field used for ordering</param>
+    /// <returns>NewsResponse</returns>
+    public async Task<NewsResponse> GetNewsAsync(
+        DateTime? startTime, 
+        DateTime? endTime, 
+        string ticker = null,
+        string order = null,
+        int? limit = null,
+        string sort = null
+    )
+    {
+        var queryParams = new Dictionary<string, string>
+        {
+            { nameof(ticker), ticker },
```

Comment:
Do we need any more checks for the optional parameters here?

Reply:
I didn't think so, the API rest have values by default when I sent a null value.
But I'd like to know what you think or what you propose.